cdap-ui/app/cdap/components/AbstractWidget/MultiStageSelector/index.tsx

```diff
@@ -58,28 +60,44 @@ const MultiStageSelectorBase: React.FC<IMultiStageSelectorProps> = ({
 }) => {
   const inputSchema = objectQuery(extraConfig, 'inputSchema');
   const inputStages = getInputStages(inputSchema);
-  const { delimiter } = widgetProps;
-  const multiSelectWidgetProps = {
-    delimiter,
-    options: inputStages,
-    showSelectionCount: false,
-    emptyPlaceholder: 'Select input stage',
-  };
+  const { delimiter, singleSelectOnly } = widgetProps;
```

Comment:
nit: if `singleSelectOnly` value is not provided set it to false here like this,

```
const { delimiter, singleSelectOnly = false } = widgetProps;
```

Reply:
done